### PR TITLE
update issue bug report template

### PR DESCRIPTION
**Pull request by Lijiaoa**
- add `Describe the bug` at the issue beginning, in this item, could summary the question and upload related image bug message

- bold `NNI version` `nnimanager.log | dispatcher.log | nnictl stdout and stderr`

- `How to reproduce it?` : some people will write something in this plate, and it's beneficial to us so I don't want to delete it.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -15,15 +19,14 @@ about: Report an issue or question while using nni instance (deployment).
...
+
 **Log message**:
- - nnimanager.log: 
- - dispatcher.log:
- - nnictl stdout and stderr:
+ - **nnimanager.log:**
+ - **dispatcher.log:**
+ - **nnictl stdout and stderr:**
  
 <!-- Where can you find the log files: [log](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director), [stdout/stderr](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout) -->
```
Comment (liuzhe-lz) [inline .github/ISSUE_TEMPLATE/bug-report.md:29]: I think it's meaningless to use markdown in comment.
Suggest change to this format:
```
<!--
Where can you find the log files:
LOG: https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director
STDOUT/STDERR: https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout
-->
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -5,8 +5,12 @@ about: Report an issue or question while using nni instance (deployment).
 
 ---
 
+**Describe the bug**:
```
Comment (liuzhe-lz) [inline .github/ISSUE_TEMPLATE/bug-report.md:8]: Suggest change "bug" to "issue".

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (Lijiaoa) [reply .github/ISSUE_TEMPLATE/bug-report.md:29]: yes

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (Lijiaoa) [reply .github/ISSUE_TEMPLATE/bug-report.md:8]: ok

**Review by QuanluZhang — APPROVED** [review]

**Comment by liuzhe-lz** [comment]
I recently found experiment config very useful.
I suggest update the template like follow.

```
**Describe the issue**:



**Environment**:
- NNI version:
- Training service (local/remote/aml/etc):
- Client OS:
- Server OS (for remote mode only):
- Python version:
- PyTorch/TensorFlow version:
- Is conda/virtualenv/venv used?:
- Is running in Docker?:



**Configuration**:
Experiment config: (remember to remove secrets!)
Search space:



**Log message**:
nnimanager.log:
dispatcher.log:
nnictl stdout and stderr:
 
<!--
Where can you find the log files:
LOG: https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director
STDOUT/STDERR: https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout
-->



**How to reproduce it?**:
```

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**